### PR TITLE
Adds One SG box to Cargo weapons vendor. SG mags are more expensive in req.

### PR DESCRIPTION
**Pull request by Naaanii**
## About The Pull Request

Adds ONE SG to the cargo weapon vendor and increases the prices of SG mags in req, making them harder to maintain without regular point income.

## Why It's Good For The Game

Allows marines to get one SG, (max 2-3 depending on the ship) giving them slightly better ability to push out at the start but also makes it harder to maintain if no points are gained.

## Changelog
:cl:
add: Added SG crate to Automated Arnaments vendor.
balance: Increased the price of SG mags in req.
/:cl:

### CODE REVIEW

**Comment by neohenry** [comment]
time for me to hack req for SG kit

**Comment by Naaanii** [comment]
> time for me to hack req for SG kit

Vendors are in marine prep though...

File: code/modules/reqs/supplypacks.dm [hunk 1]
```diff
@@ -567,7 +567,7 @@ AMMO
 /datum/supply_packs/ammo/smartmachinegun
 	name = "T-29 smartmachinegun ammo"
 	contains = list(/obj/item/ammo_magazine/standard_smartmachinegun)
-	cost = 5
+	cost = 10
```
Comment (Pariah919) [inline code/modules/reqs/supplypacks.dm:570]:
```suggestion
	cost = 7
```

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 2]
```diff
@@ -205,6 +205,7 @@
 		/obj/item/storage/box/m94 = 30,
 		/obj/item/storage/box/m94/cas = 10,
 		/obj/item/storage/box/recoilless_system = 1,
+		/obj/item/storage/box/t26_system = 1,
```
Comment (Pariah919) [inline code/game/objects/machinery/vending/marine_vending.dm:208]:
```suggestion
		/obj/item/storage/box/t26_system = 1,
```
put these in the operational supply locker so there's a better tight grasp on the amount of these per round, should really go with all spec guns or whatever power crate garbage we make
I'll do it one day when i'm not tired and lazy

**Review by Pariah919 — COMMENTED** [review]
it is 2 am and i am tired

File: code/game/objects/machinery/vending/marine_vending.dm (hunk 2 above)
Comment (Naaanii) [reply code/game/objects/machinery/vending/marine_vending.dm:208]: > put these in the operational supply locker so there's a better tight grasp on the amount of these per round, should really go with all spec guns or whatever power crate garbage we make
> I'll do it one day when i'm not tired and lazy

I intend to change the amount of vendors on both maps.
As of now, the PoS will have 3 and Sulaco will have 2 with the current iteration.

Basically you get the same amount of SGs as the number of RRs on a ship.

**Comment by SpaceLoveSs13** [comment]
YEEEEEEESSS!

**Comment by Losenis** [comment]
Should this not wait till we know for sure if SGer role removal is reverted or not?

**Comment by DerrickChamberlain** [comment]
We should just add the SG role back, it would make all this brain work of thinking how to balance it go away.

File: code/game/objects/machinery/vending/marine_vending.dm (hunk 2 above)
Comment (Pariah919) [reply code/game/objects/machinery/vending/marine_vending.dm:208]: this is dumb and stupid jsut put them in the operational supply vendor so its more finely tuned and less obfsucated codewise
I'll do that after this pr

**Comment by neohenry** [comment]
so when po's vendors in PoS get qdel?

**Comment by TiviPlus** [comment]
I there are too many low effort balance prs and I am closing this pr
Please put more effort into your PR body or into the design of this balance change

**Pull request closed without merging**